tests/test_assignment_type.py

```diff
@@ -1,21 +1,27 @@
 import os
+import zipfile
 
-import pytest
-
-import fasttrips
 from fasttrips import Run
 
 
 def test_deterministic():
 
-    EXAMPLES_DIR   = os.path.join(os.getcwd(),"fasttrips","Examples","test_scenario")
+    EXAMPLES_DIR   = os.path.join(os.getcwd(),"fasttrips","Examples")
 
-    INPUT_NETWORKS = os.path.join(EXAMPLES_DIR,"network")
-    INPUT_DEMAND   = os.path.join(EXAMPLES_DIR,"demand_reg")
+    INPUT_NETWORKS = os.path.join(EXAMPLES_DIR,"networks")
+    INPUT_DEMAND   = os.path.join(EXAMPLES_DIR, 'demand', "demand_reg")
     OUTPUT_DIR     = os.path.join(EXAMPLES_DIR,"output")
 
+    scenario_dir = os.path.join(INPUT_NETWORKS, 'simple')
```

Comment:
A scenario file would specify everything...this is just specifying what network to use.  Suggest deleting this intermediate step [ and as before, not using zip files ]

Reply:
This test is a little messy, and not fully implemented. Attempting to identify the bare minimum necessary to run the C++ wrapper code and isolate assignment.